opencti-platform/opencti-front/src/private/components/common/files/FileManager.jsx

```diff
@@ -323,6 +336,19 @@ const FileManager = ({
                       })}
                     </Field>
                   )}
+                  {selectedConnector?.connector_scope?.includes('text/csv')
+                    && (
+                      <>
+                        <ObjectMarkingField
+                          name="objectMarking"
+                          style={fieldSpacingContainerStyle}
+                        />
+                        <DialogContentText>
+                          {t('Marking definitions to use by the csv mapper...')}
```

Comment:
question: shouldn't we use a longer name function for text now ?

Reply:
In this component the translation function is taken as a prop instead of using the hook `useTranslate` (because it is a class component so it cannot use hooks), a prop called `t`. The way this component is called it is not trivial to change this name. And I don't think it should eb done in this PR but yes you are right that if possible we should use `t_i18n` as name of the translation function